scripts/publish.sh

```diff
@@ -42,7 +42,8 @@ fi
 if [ "$PRODUCTION" = "1" ] && [ "$CHANNEL" = "dev" ]; then
     echo "Sync with ${GITHUB_REF} and push to it"
     git pull github "${GITHUB_REF}" --ff-only
-    git push github HEAD:"${GITHUB_REF}"
+    git tag -a "v$NEXT_EXTENSION_VERSION-dev" -m "v$NEXT_EXTENSION_VERSION-dev" -m "Prisma version: $PRISMA_VERSION"
```

Comment:
Won't the suffix mean that these tags are mixed into the normal tags instead of listed next to each other? (e.g. if you list the git tags)

Reply:
We could also use folders for this by the way, some client use this to group/hide all the individual tags.

Reply:
> Won't the suffix mean that these tags are mixed into the normal tags instead of listed next to each other? (e.g. if you list the git tags)

Shouldn't that list be chronological? You can also filter tags when listing (from the CLI)

> We could also use folders for this by the way, some client use this to group/hide all the individual tags.

Do you mean `dev/<version>` for unstable and `<version>` for stable?

Reply:
> Do you mean `dev/<version>` for unstable and `<version>` for stable?

That is what I would suggest.